### PR TITLE
feat: use ipfs-webui@4.3.2

### PR DESCRIPTION
**Pull request by SgtPooki**
- https://github.com/ipfs/ipfs-webui/releases/tag/v4.3.1
- https://github.com/ipfs/ipfs-webui/releases/tag/v4.3.2

### CODE REVIEW

**Review by lidel — CHANGES_REQUESTED** [review]
Ok, I think things with https://github.com/ipfs/ipfs-webui/issues/2266 are worse than I thought, it IS a regression because local Kubo  is not used at all by IPLD Explorer.


Just try opening IPLD Explorer's  Wikipedia example on empty browser profile (without Helia caching a block):


| v4.3.0 only hits local Kubo | v4.3.1 hits random gateways and not local Kubo  | 
| ---- | ---- |
| old-2024-09-23_19-42 | new-2024-09-23_19-44 |

So we can't update ipfs-webui without fixing https://github.com/ipfs/ipfs-webui/issues/2266 first.

**Comment by SgtPooki** [comment]
I will work on that bug asap

File: core/corehttp/webui.go [hunk 1]
```diff
@@ -1,11 +1,12 @@
 package corehttp
 
 // WebUI version confirmed to work with this Kubo version
-const WebUIPath = "/ipfs/bafybeihatzsgposbr3hrngo42yckdyqcc56yean2rynnwpzxstvdlphxf4" // v4.3.0
+const WebUIPath = "/ipfs/bafybeideglc722hiwhsy4kiyl2fivf5lr6wozy2iuixtgzkvl3v4hasaty" // v4.3.1
```
Comment (lidel) [inline core/corehttp/webui.go:4]:
```suggestion
const WebUIPath = "/ipfs/bafybeif6abowqcavbkz243biyh7pde7ick5kkwwytrh7pd2hkbtuqysjxy" // v4.3.2
```

**Review by lidel — APPROVED** [review]
switched to [v4.3.2](https://github.com/ipfs/ipfs-webui/releases/tag/v4.3.2) lgtm, thank you @SgtPooki

**Pull request merged**